modules/help_editor/jsx/help_editor.js

```diff
@@ -0,0 +1,135 @@
+import FilterForm from 'FilterForm';
+import formatColumn from './columnFormatter';
+
+/**
+ * Help Editor Archive Page.
+ *
+ * Serves as an entry-point to the module, rendering the whole react
+ * component page on load.
+ *
+ * Renders Help Editor main page consisting of FilterTable and
+ * DataTable components.
+ *
+ * @author Alex Ilea
+ * @version 1.0.0
+ *
+ * */
+class HelpEditor extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoaded: false,
+      filter: {}
+    };
+
+    // Bind component instance to custom methods
+    this.fetchData = this.fetchData.bind(this);
+    this.updateFilter = this.updateFilter.bind(this);
+    this.resetFilters = this.resetFilters.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  /**
+   * Retrive data from the provided URL and save it in state
+   * Additionaly add hiddenHeaders to global loris vairable
+   * for easy access by columnFormatter.
+   */
+  fetchData() {
+    $.ajax(this.props.DataURL, {
+      method: "GET",
+      dataType: 'json',
+      success: function(data) {
+        loris.hiddenHeaders = data.hiddenHeaders ? data.hiddenHeaders : [];
```

Comment:
I'm pretty sure @driusan had mentioned it at the react workshop, but I don't think there's any reason to be using the `loris` global variable here.

Reply:
Clarification: 
When things were starting to be reactified, hidden headers were set using the `loris` global variable. The same one that you use to get the baseURL (`loris.baseURL`). The reason for this is that the `columnFormatter` function was in a separate file and needed to access the `hiddenHeaders`. However, it's completely unnecessary that the `formatColumn` function be in a separate file. Therefore, on this line in particular, there's no need to set the value to the `loris` variable. I would suggest moving the `formatColumn` file into this function, and setting `hiddenHeaders` as a state.